Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -27,6 +27,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 @interface SPTDataLoaderServiceDefaultSessionSelector ()
 
 @property (nonatomic, strong, readonly) NSURLSessionConfiguration *configuration;
+@property (nonatomic, strong, readonly, nullable) NSURLSessionConfiguration *backgroundConfiguration;
+@property (nonatomic, strong, readwrite, nullable) NSURLSession *backgroundSession;
```

Comment:
I guess this can be move where the other Session are. Line `40` and `41`

Reply:
I could move it there, but what would be the benefit of that?

Reply:
I should make it readonly, actually, so I can add an instance var there instead of synthesizing.